temp.py

```diff
@@ -0,0 +1,9 @@
+from idom import component
+
+
+@component
+def temp(x: int, y: int) -> None:
+    return None
+
+
+temp(*[1, 2])
```

Comment:
We might want to put `temp.py` into the gitignore if you frequently use it for testing